syntax = "proto3";

package server_diva;

service server_diva {
    rpc request_frame_path(query_statement) returns (directory) {}
}

message query_statement {
    string name = 1;
}

message directory {
    string directory_path = 1;
}

